src/MutatorProvider.php

```diff
@@ -63,45 +64,51 @@ public function registerMutators(array $mutators)
     }
 
     /**
-     * @param string $offset
+     * @param  string  $offset
      * @return bool
      */
+    #[\ReturnTypeWillChange]
     public function offsetExists($offset)
     {
         return $this->exists($offset);
     }
 
     /**
-     * @param string $offset
+     * @param  string  $offset
      * @return \Weebly\Mutate\Mutators\MutatorContract
+     *
      * @throws \Weebly\Mutate\Exceptions\MutatorNotFoundException
      */
+    #[\ReturnTypeWillChange]
     public function offsetGet($offset)
     {
         return $this->get($offset);
     }
 
     /**
-     * @param string $offset
-     * @param mixed  $value
+     * @param  string  $offset
+     * @param  mixed  $value
      * @return \Weebly\Mutate\MutatorProvider
      */
+    #[\ReturnTypeWillChange]
     public function offsetSet($offset, $value)
     {
         return $this->set($offset, $value);
     }
 
     /**
-     * @param string $offset
+     * @param  string  $offset
      */
+    #[\ReturnTypeWillChange]
```

Comment:
What behavior are we driving from this? The `composer.json` still says php 7.x is OK. On 7, this will be a `comment` does that cause a problem?

Reply:
Adding this attribute is necessary to prevent these errors in PHPStan (also, PHP will throw an error when executing this code in PHP 8.1 if this attribute is not added or the return type is not set correctly.

```
 ------ -------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Line   src/MutatorProvider.php
 ------ -------------------------------------------------------------------------------------------------------------------------------------------------------------------
  69     Return type mixed of method Weebly\Mutate\MutatorProvider::offsetExists() is not covariant with tentative return type bool of method ArrayAccess::offsetExists().
         💡 Make it covariant, or use the #[\ReturnTypeWillChange] attribute to temporarily suppress the error.
  89     Return type mixed of method Weebly\Mutate\MutatorProvider::offsetSet() is not covariant with tentative return type void of method ArrayAccess::offsetSet().
         💡 Make it covariant, or use the #[\ReturnTypeWillChange] attribute to temporarily suppress the error.
  97     Return type mixed of method Weebly\Mutate\MutatorProvider::offsetUnset() is not covariant with tentative return type void of method ArrayAccess::offsetUnset().
         💡 Make it covariant, or use the #[\ReturnTypeWillChange] attribute to temporarily suppress the error.
 ------ -------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

In older PHP versions, since it is `#` prefixed, it will be ignored as a comment and will be perfectly fine, since it doesn't apply anyway.